.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "packages/contracts/lib/forge-std"]
 	path = packages/contracts/lib/forge-std
-	url = https://github.com/foundry-rs/forge-std
+	url = https://github.com/sphinx-labs/forge-std
```

Comment:
I've tested that the switch to this fork works smoothly and [created a ticket to switch back once the next release for `forge-std` happens.](https://linear.app/chugsplash/issue/CHU-640/switch-back-to-official-forge-std-version)